src/Components/Movies.js

```diff
@@ -0,0 +1,51 @@
+import axios from "axios";
+import { Component } from "react";
+import "./Movies.css";
+
+class Movies extends Component {
+  state = { movies: [], selectedMovie: {} };
+
+  fetchMovies = async () => {
+    try {
+      const res = await axios.get("https://ghibliapi.herokuapp.com/films/");
+      this.setState({ movies: res.data });
+      // debugger
+    } catch (error) {
+      this.setState({ movies: [] });
+    }
+  };
+
+  componentDidMount() {
+    this.fetchMovies();
+  }
+
+  selectMovie = async (e) => {
+    try {
+      const res = await axios.get(`https://ghibliapi.herokuapp.com/films/${e.target.value}`);
+      this.setState({ selectedMovie: res.data});
+    } catch (error) {
+      this.setState({ selectedMovie: {} });
+    }
+ 
+  };
+
+  render() {
+    const { movies, selectedMovie } = this.state;
+    return (
+      <section className="MovieContainer">
+        <h1 style={{ color: "yellow" }}>Select a Movie</h1>
+        <select id="" onChange={this.selectMovie}>
```

Comment:
Best practice to have controlled components. Select should have a value that mirrors state.